docs/quickstart-gce.md

```diff
@@ -0,0 +1,246 @@
+# How To Install Kubernetes On GCE Cluster Using KubeOne
+
+In this quick start we're going to show how to get started with KubeOne on GCE.
+We'll cover how to create the needed infrastructure using our example terraform
+configuration and then install Kubernetes. Finally, we're going to show how to
+destroy the cluster along with the infrastructure.
+
+As a result, you'll get Kubernetes 1.13.5 High-Available (HA) clusters with
```

Comment:
Maybe we can use 1.14.0 as v0.5.0 is going to support it. :)

Reply:
👍